src/server.h

```diff
@@ -334,6 +342,9 @@ class Server : public con::PeerHandler, public MapEventReceiver,
 	virtual bool registerModStorage(ModMetadata *storage);
 	virtual void unregisterModStorage(const std::string &name);
 
+	void SendMinimapModes(session_t peer_id, std::vector<MinimapMode> modes,
+			size_t wanted_mode);
+
```

Comment:
(not a requirement)

Would you please be so nice and move this line after `SendPlayerFov` to have similar functions together (perhaps in the cpp file as well).

Reply:
Done